pkg/configuration/constants.go

```diff
@@ -17,7 +17,9 @@ const (
 	WORKFLOW_USE_STDIO             string = "wflstdio"
 	RAW_CMD_ARGS                   string = "raw_cmd_args"
 	WEB_APP_URL                    string = "internal_snyk_app"
+	INPUT_DIRECTORY                string = "targetDirectory"
+	ADD_TRUSTED_CA_FILE            string = "internal_additional_trusted_ca_file" // pem file location containing additional CAs to trust
```

Comment:
If this description is correct, the initialization with the env var name is not correct - it should be resolved before being added.